token/services/network/common/rws/translator/translator.go

```diff
@@ -161,11 +148,36 @@ func (w *Translator) QueryTokens(ids []*token.ID) ([][]byte, error) {
 }
 
 func (w *Translator) GetTransferMetadataSubKey(k string) (string, error) {
-	return keys.GetTransferMetadataSubKey(k)
+	return w.KeyTranslator.GetTransferMetadataSubKey(k)
 }
 
-func (w *Translator) AreTokensSpent(id []string, graphHiding bool) ([]bool, error) {
-	return w.areTokensSpent(id, graphHiding)
+func (w *Translator) AreTokensSpent(ids []string, graphHiding bool) ([]bool, error) {
```

Comment:
Maybe we could explain what makes `AreTokensSpent` different from the checks in `CheckTransfer`